lib/app.js

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import express from 'express';
+import {router} from './routes/index.js';
+
+export const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({extended: false}));
```

Comment:
Do we need to support url-encoded stuff? If not, let's remove it.

Reply:
Removed [129f265](https://github.com/digitalbazaar/vc-holder-http/pull/1/commits/129f265ad2da2dfbaeb4e09544994dfe06290d09)